client/jetpack-connect/test/plans.js

```diff
@@ -30,21 +30,23 @@ describe( 'Plans', () => {
 		expect( wrapper.find( QueryPlans ) ).toHaveLength( 1 );
 	} );
 
-	test( 'should render empty with unknown plan', () => {
+	test( 'should render placeholder with unknown plan', () => {
 		const wrapper = shallow( <Plans { ...DEFAULT_PROPS } hasPlan={ null } /> );
 
+		expect( wrapper ).toMatchSnapshot();
 		expect( wrapper.find( PlansGrid ) ).toHaveLength( 0 );
 		expect( wrapper.find( QueryPlans ) ).toHaveLength( 1 );
 	} );
 
-	test( 'should render empty with unknown Atomic', () => {
+	test( 'should render placeholder with unknown Atomic', () => {
 		const wrapper = shallow( <Plans { ...DEFAULT_PROPS } isAutomatedTransfer={ null } /> );
 
+		expect( wrapper ).toMatchSnapshot();
 		expect( wrapper.find( PlansGrid ) ).toHaveLength( 0 );
 		expect( wrapper.find( QueryPlans ) ).toHaveLength( 1 );
 	} );
 
-	test( 'should render empty with a paid plan', () => {
+	test( 'should render placeholder with a paid plan', () => {
```

Comment:
I think we should structure these tests a bit differently now that we have a different method for `shouldShowPlaceholder`:

* We can test the implementation of `shouldShowPlaceholder` by checking the return value under different circumstances [like here](https://github.com/Automattic/wp-calypso/blob/ecd902ee6f36ca22becf94f14e2a512cdfcbd4de/client/jetpack-connect/test/auth-logged-in-form.js#L68-L74).
* We can test the component render based on return of `shouldShowPlaceholder` by mocking it [like we do here](https://github.com/Automattic/wp-calypso/blob/ecd902ee6f36ca22becf94f14e2a512cdfcbd4de/client/jetpack-connect/test/auth-logged-in-form.js#L87-L111).